docs/en/integrations/data-visualization/powerbi-and-clickhouse.md

```diff
@@ -1,168 +1,209 @@
 ---
 sidebar_label: Power BI
 slug: /en/integrations/powerbi
-keywords: [clickhouse, powerbi, connect, integrate, ui]
+keywords: [ clickhouse, powerbi, connect, integrate, ui ]
 description: Microsoft Power BI is an interactive data visualization software product developed by Microsoft with a primary focus on business intelligence.
 ---
+
 import ConnectionDetails from '@site/docs/en/_snippets/_gather_your_details_http.mdx';
 
 # Power BI
 
-Power BI can load the data from ClickHouse Cloud or on-premise deployment using either the [ODBC driver](https://github.com/ClickHouse/clickhouse-odbc) or [ClickHouse Native connector](https://github.com/ClickHouse/power-bi-clickhouse). Both methods support Load mode, but the latter also supports Direct Query mode which eliminates the necessity to load the entire table.
+Power BI can load the data from ClickHouse Cloud or on-premise deployment using either
```

Comment:
Did you mean to have break lines in this paragraph 12-13

Reply:
Yes, it is rendered correctly